.circleci/config.yml

```diff
@@ -356,6 +401,7 @@ workflows:
       - ui-build-test
       - api-bigquery-test
       - api-integration-test
+      - puppeteer-e2e-test # to test Puppeteer tests start and run. To be removed before PR merge.
```

Comment:
Remove this line. Uncomment line #414 - 419 before PR merge.

Reply:
Would prefer we don't do that, this will generate noise on unrelated PRs.